docs/reference/commandline/dockerd.md

```diff
@@ -17,73 +17,74 @@ keywords: "container, daemon, runtime"
 # daemon
 
 ```markdown
-Usage: dockerd [OPTIONS]
+Usage:	dockerd COMMAND
```

Comment:
Yep I just pasted the output of `--help` since we don't have an automatic way to generate these from the code.